python/cudf/cudf/core/series.py

```diff
@@ -815,6 +817,101 @@ def __array_function__(self, func, types, args, kwargs):
         else:
             return NotImplemented
 
+    def map(self, arg, na_action=None) -> "Series":
+        """
+        Map values of Series according to input correspondence.
+
+        Used for substituting each value in a Series with another value,
+        that may be derived from a function, a ``dict`` or
+        a :class:`Series`.
```

Comment:
I'm not sure XD I'll double-check and fix it if it isn't!

Reply:
The :class:`Series` is supposed to link to the Series class. I'm not sure if it's doing this correctly. I've replaced it with double ticks just in case it's not